freqtrade/leverage/liquidation_price.py

```diff
@@ -0,0 +1,106 @@
+from freqtrade.enums import Collateral, TradingMode
+from freqtrade.exceptions import OperationalException
+
+
+def liquidation_price(
+    exchange_name: str,
+    trading_mode: TradingMode,
+    ** k
+):
+
```

Comment:
> Please don't use **kwargs in methods where the variable is mandatory, or where it's not a user interface you might wanna expand in the future.
I know some libraries use **kwargs extensively, but it's usually very bad practice and leads to bugs.
Using explicit parameters (eventually with defaults if they're not always relevant) will allow mypy to help you detect potential problems.

Different exchanges and trading_mode's might need different arguments, I'm not sure yet though

Also, Collateral isn't required if trading_mode is SPOT, but I suppose I could make it an Optional

Reply:
as mentioned above - specify all parameters explicitly. Parameters that are not needed in all cases can be Optional (with a default of None or so).